releases/pritunl.yaml

```
@@ -93,7 +95,7 @@ releases:
         annotations:
           kubernetes.io/ingress.class: {{ env "PRITUNL_INGRESS_CLASS" | default "nginx" }}
         hosts:
-          {{ env "PRITUNL_UI_EXTERNAL_HOST" }}: /
+          {{ env "PRITUNL_UI_EXTERNAL_HOST" | default "pritunl" }}: /
```

I don't like providing a default host name if the user is creating an ingress. Let's do this instead:

```suggestion
          {{ required "Must specify external host name for Ingress" (or (not (eq (env "PRITUNL_INGRESS_ENABLED") "true")) "PRITUNL_UI_EXTERNAL_HOST") }}: /
```

---

Thanks for the pointer. I was looking for something like that but couldn't find example.

---

This is neat.

---

@Nuru `required` was added in `helmfile` https://github.com/roboll/helmfile/commit/7d11f5dedc6b2c26a29af79c2c601a6388713d5a

---

@Nuru these are excellent recommendations. Unfortunately, the scope impact with a customer exploded with these requested changes (after some scoping work), so we need to be pragmatic and first address a few upgrades before doing this change. 

Will reject this review and create an issue for it so we come back to it.

---

@osterman OK, I agree that this is not worth a big increase in scope, but curious about what changed that is not backward compatible. Also, in case it matters, `required` was added to helmfile in [v0.109.0](https://github.com/roboll/helmfile/releases/tag/v0.109.0).

---

I think they are only 7 minor releases behind.   Thought it was a lot more.